controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -1661,3 +1762,214 @@ int cli_bml::steering_client_measure(uint32_t steeringGroupIndex, const std::str
     return 0;
 }
 #endif //BEEROCKS_RDKB
+
+int cli_bml::set_dcs_continuous_scan_enable(const std::string &radio_mac, int8_t enable)
+{
+    std::cout << "set_dcs_continuous_scan_enable" << std::endl;
+
+    int ret = bml_set_dcs_continuous_scan_enable(ctx, radio_mac.c_str(), enable);
+
+    printBmlReturnVals("bml_set_dcs_continuous_scan_enable", ret);
+
+    return 0;
+}
+
+int cli_bml::get_dcs_continuous_scan_enable(const std::string &radio_mac)
+{
+    std::cout << "get_dcs_continuous_scan_enable" << std::endl;
+
+    int enable = -1;
+    int ret    = bml_get_dcs_continuous_scan_enable(ctx, radio_mac.c_str(), &enable);
+
+    if (ret == BML_RET_OK) {
+        std::cout << "dcs-continuous-scan-enable=" << ((enable == 1) ? "True" : "False")
+                  << std::endl;
+    }
+
+    printBmlReturnVals("bml_get_dcs_continuous_scan_enable", ret);
+
+    return 0;
+}
+
+int cli_bml::set_dcs_continuous_scan_params(const std::string &radio_mac, int dwell_time,
+                                            int interval_time, const std::string &channel_pool,
+                                            int channel_pool_size)
+{
+    std::cout << "set_dcs_continuous_scan_params" << std::endl;
+
+    int ret = -1;
+    if (channel_pool.length() == 0 || channel_pool_size == BML_DCS_INVALID_PARAM) {
+        ret = bml_set_dcs_continuous_scan_params(ctx, radio_mac.c_str(), dwell_time, interval_time,
+                                                 nullptr, BML_DCS_INVALID_PARAM);
+    } else {
+        auto channels      = string_utils::str_split(channel_pool, ',');
+        auto channels_size = channels.size();
+
+        if (channels_size > BML_DCS_MAX_CHANNEL_POOL_SIZE) {
+            std::cout << "size of channel_pool is too big. size=" << channels_size << std::endl;
+            return -1;
+        }
+        if (channels_size > 0 && channels_size != (size_t)channel_pool_size) {
+            std::cout << "size of channel_pool(" << channels_size << ") != channel_pool_size("
+                      << channel_pool_size << ")" << std::endl;
+            return -1;
+        }
+
+        unsigned int channel_pool_arr[BML_DCS_MAX_CHANNEL_POOL_SIZE] = {0};
+
+        for (size_t i = 0; i < channels_size; i++) {
+            channel_pool_arr[i] = beerocks::string_utils::stoi(channels[i]);
+        }
+
+        ret = bml_set_dcs_continuous_scan_params(ctx, radio_mac.c_str(), dwell_time, interval_time,
+                                                 channel_pool_arr, channel_pool_size);
+    }
+    printBmlReturnVals("bml_set_dcs_continuous_scan_params", ret);
+
+    return 0;
+}
+
+int cli_bml::get_dcs_continuous_scan_params(const std::string &radio_mac)
+{
+    std::cout << "get_dcs_continuous_scan_params" << std::endl;
```

Comment:
You should probably be using __func__ instead of a literal string to avoid copy & paste errors. This way maintenance is easier (if method name changes, there is no need to change anything else). 
This is something to be considered in all the methods in this file.

Reply:
fixed